openfisca_us/variables/gov/states/md/tax/income/credits/poverty_line/md_poverty_line_credit.py

```diff
@@ -0,0 +1,40 @@
+from openfisca_us.model_api import *
+
+
+class md_poverty_line_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD Poverty Line Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://law.justia.com/codes/maryland/2021/tax-general/title-10/subtitle-7/section-10-709/"
+
+    def formula(tax_unit, period, parameters):
+        # Earlier portions of the law define eligibility.
+        eligible = tax_unit("is_eligible_md_poverty_line_credit", period)
+        # (c)    Except as provided in subsection (e) of this section, the
+        # credit allowed against the State income tax under subsection (b)(1)
+        # of this section equals the lesser of:
+        # (1) the State income tax determined after subtracting the credit
+        # allowed under § 10–704(b)(1) of this subtitle; or
+        income_tax_before_credits = tax_unit(
+            "md_income_tax_before_credits", period
+        )
+        md_non_refundable_eitc = tax_unit(
```

Comment:
Add a comment linking to the form that this is non-refundable EITC, since the law is ambiguous